elasticsearch/docker/templates/Dockerfile.j2

```diff
@@ -191,7 +193,7 @@ RUN chgrp 0 /usr/local/bin/docker-entrypoint.sh && \
     chmod g=u /etc/passwd && \
     chmod 0775 /usr/local/bin/docker-entrypoint.sh
 
-EXPOSE 9200 9300 9600
+EXPOSE 9200 9300 9600 9650
```

Comment:
In the discussion meeting, you guys mentioned there is no new exposed port. what's this new one for?